vips/vip-liquidator.ts

```diff
@@ -0,0 +1,129 @@
+        params: [LIQUIDATOR, "restrictLiquidation(address)", FAST_TRACK_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "restrictLiquidation(address)", CRITICAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "unrestrictLiquidation(address)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "addToAllowlist(address,address)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "addToAllowlist(address,address)", FAST_TRACK_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "addToAllowlist(address,address)", CRITICAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "removeFromAllowlist(address,address)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "setTreasuryPercent(uint256)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "setMinLiquidatableVAI(uint256)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "pauseForceVAILiquidate()", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "pauseForceVAILiquidate()", FAST_TRACK_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "pauseForceVAILiquidate()", CRITICAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "resumeForceVAILiquidate()", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "resumeForceVAILiquidate()", FAST_TRACK_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "resumeForceVAILiquidate()", CRITICAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "setPendingRedeemChunkLength(uint256)", NORMAL_TIMELOCK],
+      },
```

Comment:
Just a reminder to add the command to upgrade the implementation of the Liquidator with the new contract. It should be something like this:

```
0x2b40B43AC5F7949905b0d2Ed9D6154a8ce06084a.upgrade("0x0870793286aaDA55D39CE7f82fb2766e8004cF43", "<TODO: new Liquidator implementation>")
```

where:
- 0x0870793286aaDA55D39CE7f82fb2766e8004cF43: Liquidator proxy
- 0x2b40B43AC5F7949905b0d2Ed9D6154a8ce06084a: ProxyAdmin contract, owned by 0x939bd8d64c0a9583a7dcea9933f7b21697ab6396
- 0x939bd8d64c0a9583a7dcea9933f7b21697ab6396: Timelock contract, used in the Venus Governance process. It allows votes for 24 hours, and then it has a delay of 48 hours until the proposed VIP is executed

@GitGuru7